cmake/sysbuild/suit.cmake

```diff
@@ -149,6 +149,57 @@ function(suit_register_post_build_commands)
   )
 endfunction()
 
+function(suit_generate_dfu_zip)
+  get_property(
+    dfu_artifacts
+    GLOBAL PROPERTY
+    SUIT_DFU_ARTIFACTS
+  )
+
+  set(script_params "")
+
+  foreach(image ${IMAGES})
+    unset(GENERATE_LOCAL_ENVELOPE)
+    sysbuild_get(GENERATE_LOCAL_ENVELOPE IMAGE ${image} VAR CONFIG_SUIT_LOCAL_ENVELOPE_GENERATE KCONFIG)
+    if(NOT DEFINED GENERATE_LOCAL_ENVELOPE)
+      continue()
+    endif()
+
+    sysbuild_get(target IMAGE ${image} VAR CONFIG_SUIT_ENVELOPE_TARGET KCONFIG)
+
+    unset(domain_app)
+    unset(domain_rad)
+    sysbuild_get(domain_app IMAGE ${image} VAR CONFIG_SOC_NRF54H20_CPUAPP KCONFIG)
+    sysbuild_get(domain_rad IMAGE ${image} VAR CONFIG_SOC_NRF54H20_CPURAD KCONFIG)
+
+    if(domain_app)
+      set(type "suit-application")
+    elseif(domain_rad)
+      set(type "suit-radio")
```

Comment:
Are there more types supported than application and radio? What with secdom, storage, sysctrl and other?

Reply:
I'll probably know the answer to that question tomorrow.

Reply:
We decided that we won't distinguish any manifest types. All artifacts now have `type="bin"`, except the root.suit which has `type="suit-envelope"`.